api-docs/v2.0/swagger.yml

```diff
@@ -6781,6 +6782,27 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Expression"
+    DictExpression:
```

Comment:
I think this is in `2.0`, but I missed the cherry-pick to update swagger.yml. Will need to fix that before we release.

Reply:
influxdata/influxdb#20708